spec/flipper/railtie_spec.rb

```diff
@@ -3,30 +3,29 @@
 require 'flipper/railtie'
 
 RSpec.describe Flipper::Railtie do
-  let(:application) do
-    app = Class.new(Rails::Application).new(
-      railties: [Flipper::Railtie],
-      ordered_railties: [Flipper::Railtie]
-    )
-    app.config.eager_load = false
-    app.config.logger = ActiveSupport::Logger.new($stdout)
-    app.run_load_hooks!
+  let(:config) do
+    Rails.application.config.flipper
```

Comment:
I don't think we run the tests multi-threaded yet, but if we do switch to that (which I'd like to do eventually to make sure we don't have any threading issues), this could fail by relying on the `Rails.application` singleton. What would you think about assigning `let(:application)`, in the before block setting `Rails.application = application`, and having `config` delegate to `application` instead of `Rails.application`?

Reply:
further simplified, given your suggestions